src/patternfly/layouts/Page/page.scss

```diff
@@ -230,25 +186,21 @@
 
 // Sidebar
 .pf-l-page__sidebar {
-  grid-area: 2 / 1;
   z-index: var(--pf-l-page__sidebar--ZIndex);
-  width: var(--pf-l-page__sidebar--Width);
+  flex: 1;
+  order: initial;
+  width: var(--pf-l-page__sidebar--md--Width); // hardcode sidebar based on design spec, must match pf-n-nav--Width
+  max-width: var(--pf-l-page__sidebar--md--Width); // max-width value set for browser to calculate max-width while transitioning. sharing value to keep consistent
```

Comment:
should width/max-width have `--md` in the var name?

Reply:
they are in the md media query so that's the right format isn't it?

Reply:
Ah, gotcha. They weren't in a MQ in that commit for some reason https://github.com/patternfly/patternfly-next/blob/77e29c174b9d0dd482e6bc21d99bb843b316fd01/src/patternfly/layouts/Page/page.scss